hassio/src/addon-store/hassio-addon-store.ts

```diff
@@ -77,6 +75,11 @@ class HassioAddonStore extends LitElement {
         .repos=${this._repos}
       ></hassio-repositories-editor>
 
+      <hassio-search-input
```

Comment:
This should get a max width.

Reply:
Done in child component `.search-container`